proto/p4/p4runtime.proto

```diff
@@ -255,10 +255,10 @@ message MeterEntry {
 // meters is not allowed and will return an error. The insertion/deletion
 // should happen as part of INSERT/DELETE on the associated table-entry.
 // For ReadRequest, the scope is defined as follows:
-// - All meter cells for all meters if meter_id = 0 (default).
-// - All meter cells for given meter_id if table_entry.match is empty.
+// - All meter cells for all tables if table_entry.table_id = 0.
+// - All meter cells of a table if table_entry.table_id is present and
+//   table_entry.match is empty.
 message DirectMeterEntry {
-  uint32 meter_id = 1;
   // The associated table entry match.
   // table_entry.action is ignored. Other fields specify the match.
   TableEntry table_entry = 2;
```

Comment:
Since we are not using this code in production yet, it is perfectly fine to restart the proto field numbers from 1 again.

Reply:
Thanks for the comment. Fixed.